question24/style.css

```diff
@@ -0,0 +1,185 @@
+body,
+div,
+dl,
+dt,
+dd,
+ul,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+pre,
+code,
+form,
+fieldset,
+legend,
+input,
+textarea,
+p,
+blockquote,
+th,
+td {
+  margin: 0;
+  padding: 0;
+}
+* {
+    box-sizing: border-box;
+}
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+img {
+    max-width: 100%;
+}
+body {
+    height: 100vh;
+    position: relative;
+    background-color: #ededd1;
+}
+.loading {
+    width: 100px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+.wrapper {
+    box-sizing: border-box;
+    width: 100%;
+    max-width: 500px;
+    margin: auto;
+    background: #fff;
+    padding: 35px 5%;
+    border-radius: 10px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translateY(-50%) translateX(-50%);
+    box-shadow: 3px 6px 10px #999;
+}
+h1 {
+    text-align: center;
+    color: #000;
+    font-size: 30px;
+    margin-bottom: 30px;
+}
+h2 {
+    text-align: center;
+    font-size: 28px;
+    margin-bottom: 25px;
+}
+.modal p {
+    font-size: 15px;
+}
+ol ol {
+    list-style: upper-alpha;
+}
+li {
+    margin-bottom: 15px;
+    list-style: none;
+}
+.submit_button {
```

Comment:
Thank you for your review! 
Actually, I was worried about this point too, but once I implemented it without. But I thought it would be better to have it, so I added it! 
fixed commit : https://github.com/csakyo/homework/pull/38/commits/5230b90a3973aeece839c9ab8f5669ce175e0f13